builtin/provisioners/file/resource_provisioner.go

```diff
@@ -22,6 +22,9 @@ func (p *ResourceProvisioner) Apply(
 		return err
 	}
 
+	log.Printf("private_ip attribute: %#v", s.Attributes["private_ip"])
+	log.Printf("connection:host attribute: %#v", s.Ephemeral.ConnInfo["host"])
```

Comment:
Can you prefix these with `[DEBUG]`? We haven't enabled filtering yet but we plan to.